Win11ThemeTest/ButtonTest.cs

```diff
@@ -0,0 +1,160 @@
+﻿using FlaUI.Core;
+using FlaUI.Core.AutomationElements;
+using FlaUI.Core.Input;
+using FlaUI.UIA3;
+using FlaUI.Core;
+using FlaUI.Core.AutomationElements;
+using FlaUI.Core.Conditions;
+using FlaUI.Core.Definitions;
+using FlaUI.Core.Input;
+using FlaUI.UIA3;
+using FlaUI.UIA3.Identifiers;
+using FlaUI.UIA3.Patterns;
+using System.Drawing;
+using System.Linq;
+using NuGet.Frameworks;
+
+
+namespace Win11ThemeTest
+{
+    public class ButtonTest
+    {
+        private Application app;
+        private Window window;
+        //  private Window window2;
+        public Window btnwindow;
+        Button testButton;
+        Button button;
+        Button disabledbutton;
+        // Button iconbutton;
+
+        public ButtonTest()
+        {
+
+            app = Application.Launch(@"..\\..\\..\\..\\TestingApplication\\bin\\Debug\\net9.0-windows\\win-x64\\TestingApplication.exe");
+
+            using (var automation = new UIA3Automation())
+            {
+                window = app.GetMainWindow(automation);
+                testButton = window.FindFirstDescendant(cf => cf.ByAutomationId("testbtn")).AsButton();
+                Mouse.Click(testButton.GetClickablePoint());
+                Wait.UntilInputIsProcessed(TimeSpan.FromMilliseconds(2000));
+                btnwindow = window.FindFirstDescendant(cf => cf.ByName("ButtonWindow")).AsWindow();
+
+            }
+
+        }
+
+        //test if button is available in window
+        [Test]
+        public void btn1_isButtonAvailable()
+        {
+            Assert.IsNotNull(btnwindow);
+            button = btnwindow.FindFirstDescendant(cf => cf.ByAutomationId("btn")).AsButton();
+            Assert.IsNotNull(button);
+        }
+
+        //test if button is clicked
+        [Test]
+        public void btn2_isClicked()
+        {
+            // button = btnwindow.FindFirstDescendant(cf => cf.ByAutomationId("btn")).AsButton();
+            button.Click();
+            Wait.UntilInputIsProcessed();
+            var popup = btnwindow.FindFirstDescendant(cf => cf.ByName("Button Clicked")).AsWindow();
+            Assert.That(popup, Is.Not.Null);
+            Button pBtn = btnwindow.FindFirstDescendant(cf => cf.ByName("OK")).AsButton();
+            pBtn.Click();
+        }
+
+        //test if button clicked with enter key
+        [Test]
+        public void btn3_isClickableWithEnterKey()
```

Comment:
Consider adding window  close after test execution - `window.Close()` and `btnwindow.Close()`.

Reply:
Added window close methods in the last test case method.

Reply:
tests are not working correctly, please check once, for close refer how harshita made changes for combobox tests.

Reply:
Added close method separately with assert.